GOVERNANCE.md

```diff
@@ -0,0 +1,219 @@
+valuable contributions to the OpenColorIO project to become a new Committer.
+
+To nominate a new Committer, open an issue in the OCIO repository, with a
+summary of the nominee's contributions.
+
+If there are no objections raised by any Committers one week after the issue is
+opened, the nomination will be considered as accepted. Should there be any
+objections against the nomination, the TSC is responsible for working with the
+individuals involved and finding a resolution. The nomination must be approved
+by the TSC, which is assumed when there are no objections from any TSC members.
+
+Prior to the public nomination, the Committer initiating it may seek feedback
+from other Committers in private channels and work with the nominee to improve
+the nominee's contribution profile, in order to make the nomination as
+frictionless as possible.
+
+If individuals making valuable contributions do not believe they have been
+considered for a nomination, they may log an issue or contact a Committer
+directly.
+
+## Technical Steering Committee
+
+A subset of the Committers forms the Technical Steering Committee (TSC), which
+has final authority over this project. As defined in the project charter, TSC
+responsibilities include, but not limited to:
+
+* Coordinating technical direction of the Project
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Committers
+* Appointing representatives to work with other open source or open standards
+communities
+* Discussions, seeking consensus, and where necessary, voting on technical
+matters relating to the code base that affect multiple projects
+* Coordinating any marketing, events, or communications regarding the project
+
+Within the TSC are three elected leadership roles to be held by its members and
+voted on annually. Any TSC member can express interest in serving in a role, or
+nominate another member to serve. There are no term limits, and one person may
+hold multiple roles simultaneously. Should a TSC member resign from a leadership
+role before their term is complete, a successor shall be elected through the
+standard nomination and voting process to complete the remainder of the term.
+The leadership roles are:
+
+* Chair: This position acts as the project manager, organizing meetings and
+providing oversight to project administration.
+
+* Chief Architect: This position makes all the final calls on design and
+technical decisions, and is responsible for avoiding "design by committee"
+pitfalls.
+
+* ASWF TAC Representative: This position represents the project at all ASWF
+(Academy Software Foundation) TAC (Technical Advisory Council) meetings.
+
+### TSC Leaders
+
+* Chair: TBD
+* Chief Architect: TBD
+* ASWF TAC Representative: TBD
+
+### TSC Members
+
+* Mark Boorer - ILM
+* Sean Cooper - DNEG
+* Michael Dolan - SPI
+* Larry Gritz - SPI
```

Comment:
I'm happy to be invited as an observer, or simply be available for advice any time you need. If consensus is that I should be on it for a while, that's fine. I just want to make sure that nobody gets the impression that SPI is engineering a baked-in disproportionate representation. The whole point of the exercise is an evolution from an SPI-sponsored project into shared community ownership.

Reply:
My 2 cents look similar to Patrick's  ...  The v2 proposal has not generated any controversy so far, so I'm guessing that if there are any votes in the coming months they are more likely to involve project management rather than color science/technical direction.  Hopefully simple consensus will always suffice, but I think your wealth of experience with OSS would make you a valuable voting member, especially during the transition period.